### ISSUE TITLE
Airheart crash

### ISSUE THREAD

**Issue by JohnMBrooks** [peterferrie/prorwts2#1]
If Airheart is run from a ProDOS volume that was remapped to a different slot (ie, 3.5" disk on GS remapped to s2,d1 by smartport ROM), the loader crashes at $C211.

**Comment by peterferrie** [comment]
yes, it's fixed in the release source, but I haven't applied it yet to the older released titles.
